e2e-tests/src/test/rewards.rs

```diff
@@ -311,3 +304,82 @@ pub fn force_new_era(config: &Config) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn change_stake_and_force_new_era(config: &Config) -> anyhow::Result<()> {
+    const MAX_DIFFERENCE: f64 = 0.07;
+
+    let node = &config.node;
+    let accounts = get_validators_keys(config);
+    let sender = accounts.first().expect("Using default accounts").to_owned();
+    let connection = SignedConnection::new(node, sender);
+
+    let sudo = get_sudo_key(config);
+    let root_connection = RootConnection::new(node, sudo);
+
+    let (reserved_members, non_reserved_members) = get_member_accounts(config);
```

Comment:
I suspect that we may need to `change_validators` here (like we recently forgot: https://github.com/Cardinal-Cryptography/aleph-node/pull/528)